library/src/main/java/com/mikepenz/materialdrawer/model/ProfileSettingDrawerItem.java

```diff
@@ -89,7 +91,7 @@ public ProfileSettingDrawerItem withName(String name) {
     }
 
     public ProfileSettingDrawerItem withDescription(String description) {
-        this.email = new StringHolder(description);
+        this.description = new StringHolder(description);
```

Comment:
why didn't you re-use the e-mail field which already existed for "description"? Or do you differ between those?